horoscope-functions.py

```diff
@@ -0,0 +1,305 @@
+import os
+import discord 
+import json
+import asyncio
+from discord.ext import commands
+from random import choice
+
+#bot token stuff; not to be messed with :linus_gun:
+token_ = "NzA5NDA3MjY4NDg3MDM3MDE5.XrllLQ.FbL2vivvNxjxPT-wOfAvH32fK4QZ"
+token = token_[:len(token_)-1]
+
+#bot which controls everything; subclass of Client
+bot = commands.Bot(command_prefix='$')
+
+#Channels where the bot works
+channels_available = ["bot-test","botspam-v2","botspam"] 
+
+#global variables
+battle_ongoing = False
+turn_count = 1
+
+
+@bot.event
+async def on_ready():
+    await bot.get_channel(703141348131471440).send("Bruh")
```

Comment:
does this need an on-ready message? the bot already sends one for the economy part

Reply:
damn, no you can remove that.